scripts/run_commands

```diff
@@ -43,7 +43,8 @@ conda update --all --yes
 conda clean -tipy
 
 # Install conda build and deployment tools.
-conda install --yes --quiet conda-build anaconda-client jinja2 setuptools git patch python=3.7
+conda install --yes --quiet bzip2 git patch
+conda install --yes --quiet python=3.7 setuptools conda-build anaconda-client
```

Comment:
one install command is better than two, so we don't solve twice

Reply:
Yes, I agree, a single command is preferable. To retain the indent of https://github.com/conda-forge/docker-images/pull/103/commits/72431c84e8027be9a0e9c8a6ab76e22eadf9b9ba on readability/maintainability, we can just use a multi-line command:
```sh
conda install --yes --quiet \
   bzip2 git patch \
   python=3.7 setuptools \
   conda-build anaconda-client
```
The comment above the command may be updated to reflect that structure, too.

----
Regarding `bzip2`, I believe https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/building-conda-packages.html#other-tools is just not accurate on this. I'm not aware that `conda-build` uses `bzip2` directly, rather than via `python` (or `libarchive`).
My best guess is that either some ancient `conda-build` version did indeed use `bzip2` directly or someone mixed up `conda-build`'s and `Miniconda`'s requirements (the latter (currently) does require `bzip2`, IIRC). IMHO, we shouldn't add it here. (@msarahan would know better, of course.)

----
+1 on the `jinja2` removal!

Reply:
Have combined the commands and split over lines as suggested.